vault/activity_log_test.go

```diff
@@ -3984,3 +3986,105 @@ func TestActivityLog_partialMonthClientCountUsingHandleQuery(t *testing.T) {
 		}
 	}
 }
+
```

Comment:
Might be useful to have a comment here (and throughout) explaining some of the logic and overall structure of the test.